proposals/stage-0/individual-membership/README.md

```diff
@@ -0,0 +1,131 @@
+
+The continued engagement model will include first-contact communications and highlights to members before public notice and gives the foundation the opportunity to convert the transactional relationship to caring members. 
+
+#### Individual Membership management system
+
+*$2100 annually*
+Migrate over old system due to lack of interoperability and maintenance.
+
+Tendenci is an open source membership management platform. $2028 per year. Cost increases to $249 per month once we hit 1000-5000 member range. Price upgrade gives us more processing power(so we could technically remain on the lower tier because it is not per-user cost and instead based off of the performance we want our users to experience). Manages signups, renewals, emails to members (currently not possible within a single system for us), news, events. Has the potential to add corporate and individual member value with the batteries-included offering, and doesn’t cost any more for these features. Example in the wild: [http://www.sfbig.org/](http://www.sfbig.org/]. Requires minimal admin IT work and an easy-to-navigate GUI that the entire $FOUNDATION internal team could access/administer along with projects members.  
+  
+#### Voting System for elections
+CIVS
+
+- Currently used by the rest of the Linux Foundation projects 
+- Free 
+- [http://civs.cs.cornell.edu/](http://civs.cs.cornell.edu/) 
+- This proposal coincides with the membership management system because the prior election system we were using, Helios, is no longer being used by other projects in the LF.
+  
+
+### Required Resources
+
+What resources would be needed to execute it?
+
+- Administrator for elections, tooling, and communications coordination for membership 
+- Tooling: membership management system and election system 
+  
+
+### Who would be responsible?
+
+The CPC would own the maintenance and sustainability of the programs with power to delegate, and would be aided by the Executive group with resources of the $FOUNDATION to execute.
+
+### How would success be measured?
+
+Given the goals outlined in this proposal,
+
+- NPS score improves by 10% after base is established 
+    - With the introduction of better feedback mechanisms, we should be able to move some detractors to passives, and some passives to promoters. 
+- Relaunch of program results in at least a 2X growth in active membership roster in first year of program. 
+
+### Why this proposal is important
+
+With the historical data we have regarding individual membership, if Board Directors wish for the Individual Membership to be representative of the larger Node.js engaged ecosystem, we know that the number and makeup of the prior votes was not statistically significant nor representative. The Individual Membership did not have a clear path, role, or definition. 
```

Comment:
I don't think I fully understand this paragraph. Is the focus on engaging just the node community with an individual membership, or is this membership for the whole $foundation?

Reply:
lol, that's a 'Tracy didn't find and replace everything'. Fixing.